spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -32,16 +32,16 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
-[[microprofile4.1-RC1]]
-=== MicroProfile 4.1-RC1 (Q32021)
-MicroProfile 4.1-RC1 is based on Jakarta EE 8. Although Jakarta EE 8 is API backwards compatible with Java EE 8, Jakarta EE replaces Java EE dependencies with Jakarta EE dependencies for all MicroProfile specifications.
+[[microprofile4.1-RC2]]
+=== MicroProfile 4.1-RC2 (Q32021)
+MicroProfile 4.1-RC2 aligns with Jakarta Contexts and Dependency Injection 2.0, Jakarta Annotations 1.3, Jakarta RESTful Web Services 2.1, Jakarta JSON Binding 1.0, and Jakarta JSON Processing 1.1.
```

Comment:
I would still include something like: (Jakarta EE 8)/(which align with Jakarta EE 8) or similar. Later sections reference Java EE 7/8.

Reply:
Thanks for the suggestion, I have pushed an update. Would this be to everyone's satisfaction?

Reply:
Actually, just use MicroProfile 4.1 without refering to -RC2 as in this case, after you have done RC2 release, you can directly do MP 4.1 release without any PR.

Reply:
Ignore my previous suggestion. I am ok with this after all.

Reply:
Well, you are right about this. It is not mandatory to include these suffixes in the text. This is how we have been doing it until now, and it does clarify things. The downside is indeed a new commit and build for removing all this when creating a final release. I'll just go with it for this release, let's revisit this topic on the next one.